src/ray/raylet/placement_group_resource_manager.h

```diff
@@ -56,6 +56,13 @@ class PlacementGroupResourceManager {
   /// \param bundle_spec: Specification of bundle whose resources will be prepared.
   virtual bool PrepareBundle(const BundleSpecification &bundle_spec) = 0;
 
+  /// Prepare resource batched, it's an encapsulation of function `PrepareBundle`.
```

Comment:
```suggestion
  /// Prepare a list of bundles. It is guaranteed that all bundles are atomically prepared (e.g., if one of bundle cannot be prepared, all bundles are failed to be prepared)
```

`ReturnBundle` should become a private API, so it is an implementation detail. No need to mention here.

Reply:
updated.